test/parallel/test-worker-message-not-serializable.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+// Flags: --expose-internals
+
+// Check that main thread handles unserializable errors in a worker thread as
+// expected.
+
+const common = require('../common');
+common.skipIfWorker();
```

Comment:
I think this is what is causing the tests to fail.

Reply:
I blame jetlag.

Fixing....